Makefile

```
@@ -49,9 +49,15 @@ lint: ## golangci-lint
 vuln: ## govulncheck
 	go tool govulncheck ./...
 
+ifeq ($(strip $(CGO_ENABLED)),0)
+RACE_OPT =
+else
+RACE_OPT = -race
+endif
```

Why would we want to use `strip` function here? 

```suggestion
ifeq ($(CGO_ENABLED),0)
RACE_OPT =
else
RACE_OPT = -race
endif
```